wandb/apis/public.py

```diff
@@ -2352,12 +2358,13 @@ def _get_item(self):
         variable_values = {
             "projectName": self.project,
             "entityName": self._entity,
-            "runQueue": self.queue_id,
+            "runQueue": self.queue_name,
             "itemId": self.id,
         }
         try:
             res = self.client.execute(query, variable_values)  # exception w/ old server
             if res["project"]["runQueue"].get("runQueueItem") is not None:
+                self._queue_id = res["project"]["runQueue"]["id"]
```

Comment:
This is a filthy gross disgusting hack to get the RunQueue ID into the QueuedRun object, this must change.

Reply:
This can be replaced by having your new pushToRunQueueByName mutation return the queue id and then we can use that in launch_add and pipe that through to the QueuedRun class. Or this wont be necessary if you add a DeleteFromRunQueueByName mutation.